pkgs.nix

```diff
@@ -1,4 +1,4 @@
 import (fetchTarball {
-  url = "https://github.com/Holo-Host/holo-nixpkgs/archive/6c5491909b4fa1a4a1d7864fa204c1e5f28a6a1f.tar.gz";
-  sha256 = "1r5d3dqfi20xxjnbzwr1i5dsr7blc1q55dha29cfvi81dz52fmqr";
+  url = "https://github.com/Holo-Host/holo-nixpkgs/archive/c4ed4fbf07d74cbbcaec5a338e062c3114b7a825.tar.gz";
+  sha256 = "0sqjcqj6c862lilxb4siwm1rxi2bp8042r1bh1k4xc7gi5c2igvf";
```

Comment:
this bumps us to v39?

Reply:
Yes, this was tied to v39 and is dependent on PR#129, so we should merge that first. :+1: